### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Sharkboy-j** [ForkIssues/TrackerWin#1666]
Is it possible to add default value setting of this action and dont ask every time when user switches to another branch or checkout specific commit
![image](https://user-images.githubusercontent.com/13855710/195576920-3824df92-0126-4a49-ae4d-4831098bb029.png)

**Comment by DanPristupov** [comment]
No, it's not possible. I think, a confirmation is required when we modify the local changes.
